test.ipynb

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Usually, we prefer to exclude Jupyter notebook from commits as they have a large footprint on the code's nature.

A good exercise would be to remove the notebook using `git rm test.ipynb`, commit said  change and push again.